### DISCUSSION TITLE
point_0712

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#267] [Roy]
"20200307T153619_20200307T153616_T17MQV"
![image](https://user-images.githubusercontent.com/54947195/147608258-06ef5683-9451-451d-922b-a78af75a3d92.png)

"20190716T153621_20190716T153624_T17MQV"
![image](https://user-images.githubusercontent.com/54947195/147608430-2834bf38-7dc8-4084-a7f0-39cde8e2a817.png)
![image](https://user-images.githubusercontent.com/54947195/147608540-3561e631-a565-44e3-aa7d-16c4453c3fe9.png)
![image](https://user-images.githubusercontent.com/54947195/147608618-6b4b850b-26d7-4d19-924b-1b8a227f3c08.png)

"20191014T153621_20191014T153622_T17MQV"
![image](https://user-images.githubusercontent.com/54947195/147608728-a5b2b3a5-37e5-4a33-b03f-6d63c399b1eb.png)

"20190211T153619_20190211T153619_T17MQV"
![image](https://user-images.githubusercontent.com/54947195/147608798-709564c6-9e34-47ff-8cb5-3c80009fc1f4.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190716T153621_20190716T153624_T17MQV"
![Captura](https://user-images.githubusercontent.com/86502862/153457916-2aa31b03-bc78-4dea-a647-0c0a314a4429.JPG)
